index.html

```diff
@@ -67,5 +67,130 @@ <h5 class="small-text">I can help you build a product , feature or website Look
             </li>             
      </ul>               
     </section>
+    <section class="works">
+        <div class="title">
+           <h2>My Recent Works</h2>
+           <div class="line"></div>
+        </div>
+        <div class="sample">
+           <img src="./assets/images/works-section.png" alt="works">
+           <h3>Multi-Post Stories</h3>
+           <p class="works-para">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+           </p>
+           <ul class="tags">
+            <li class="tag-item">CSS</li>
+            <li class="tag-item">HTML</li>
+            <li class="tag-item">Bootstrap</li>
+            <li class="tag-item">Ruby</li>
+           </ul>
```

Comment:
Great job using an ordered list for this section :clap: 

- [ ] Kindly make sure that the list items have the correct padding so that they can fit within the container properly. Currently they appear to be moving a little out of the container which then makes the project not match the over all desired margins and padding of left and right. Kindly have a look at it.

**This is how I am seeing it right now:**

![image](https://user-images.githubusercontent.com/86318284/185756425-2011bfdd-de96-478d-9358-fb38ccfbc733.png)

**This is how it is supposed to look like according to the provided Figma design:**

![image](https://user-images.githubusercontent.com/86318284/185756447-b3c0b870-932e-4706-8f02-fb13eef91b44.png)